src/components/content/heroes/aurora/weather-easter-egg/snow.tsx

```diff
@@ -0,0 +1,33 @@
+import PrecipitationEffect, { PrecipitationType } from './precipitation-effect';
+import React, { FC } from 'react';
+import { DefaultFlares, DefaultFlaresDark } from '../default-flares';
+import styled from 'styled-components';
+
+export const AuroraSnowyFlareColor = styled.div`
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background: linear-gradient(0deg, #4943cb99 0%, #4d79ff00 90%);
+  position: absolute;
+`;
+
+interface SnowProps {
+  amount: number;
+}
+
+export const Snow: FC<SnowProps> = ({ amount }) => {
```

Comment:
Love it 🤩
![CleanShot 2023-07-20 at 09 27 30](https://github.com/satellytes/satellytes.com/assets/1701755/2ecdbd78-525d-4b29-8adf-c3cc4756ae25)